web_src/css/themes/theme-arc-green.css

```diff
@@ -180,8 +180,8 @@
   --color-input-toggle-background: #454a57;
   --color-input-border: #454a57;
   --color-input-border-hover: #505667;
-  --color-navbar: #2a2e3a;
```

Comment:
Just a question, since the names have been changed, will third party themes be broken?

Reply:
Yes, they need to do the rename as well. I try to avoid such breaks, but in this case it was necessary because otherwise the naming would have been very confusing.